tests/unit/components/sl-modal-header-test.js

```diff
@@ -0,0 +1,56 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import Ember from 'ember';
+
+moduleForComponent('sl-modal-header', 'Unit | Component | sl modal header', {
+    unit: true
+});
+
+test( 'It renders', function( assert ) {
+    assert.expect(2);
+
+    let component = this.subject();
+    assert.equal( component._state, 'preRender' );
+
+    this.render();
+    assert.equal( component._state, 'inDOM' );
+});
+
+test( 'Modal header class exists on child element', function( assert ) {
+    let component = this.subject();
+
+    assert.equal( this.$().find( '.modal-header' ).length, 1 );
+});
+
+test( 'Close button exists', function( assert ) {
+    let component = this.subject();
+
+    assert.equal( this.$().find( '.close' ).length, 1 );
```

Comment:
Should we be more specific in our selector, like in footer test, and look for button element as well?